### ISSUE TITLE
deprecation of 'raDegToString' from 'ginga.util.wcs' 

### ISSUE THREAD

**Issue by eriniocentric** [astropy/astrowidgets#158]
Hi astrowidgets team, 

I've just updated ginga it looks like there is a function change for 'raDegToString' from 'ginga.util.wcs' to 'ra_deg_to_str', 'dec_deg_to_str'. I'm getting this error.

Thanks
Erin
```
File /opt/conda/lib/python3.10/site-packages/astrowidgets/__init__.py:9, in <module>
      6 from ._astropy_init import *
      7 # ----------------------------------------------------------------------------
----> 9 from .core import *

File /opt/conda/lib/python3.10/site-packages/astrowidgets/core.py:20, in <module>
     18 from ginga.canvas.CanvasObject import drawCatalog
     19 from ginga.web.jupyterw.ImageViewJpw import EnhancedCanvasView
---> 20 from ginga.util.wcs import raDegToString, decDegToString
     22 __all__ = ['ImageWidget']
     24 # Allowed locations for cursor display

ImportError: cannot import name 'raDegToString' from 'ginga.util.wcs' (/opt/conda/lib/python3.10/site-packages/ginga/util/wcs.py)
```

**Comment by pllim** [comment]
Hi @eriniocentric , I cannot reproduce this error. Please tell me:

1. Does the error go away if you use a dev version of astrowidgets?
2. What exactly is your Ginga version and how did you install it?

**Comment by pllim** [comment]
Looks like this has been fixed in dev, so maybe time to "release", @mwcraig ?

https://github.com/astropy/astrowidgets/blob/3382c7e2a80eabb76fd802779af4ea85d5ef6837/astrowidgets/core.py#L22

**Comment by eriniocentric** [comment]
1. Yes I just did a pip -e install from https://github.com/astropy/astrowidgets.git and the bug is fixed 
2.  I pip install through pypi
astrowidgets==0.2.0
ginga==4.0.1

**Comment by pllim** [comment]
0.2.0 was from 2 years ago... 😬

**Comment by pllim** [comment]
I just released 0.3.0.

Thanks for reporting this!

**Comment by eriniocentric** [comment]
Thanks!
